rubocop/cop/primer/system_argument_instead_of_class.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rubocop"
+require "primer/classify/utilities"
+require "primer/view_components/statuses"
+
+module RuboCop
+  module Cop
+    module Primer
+      # This cop ensures that components use System Arguments instead of CSS classes.
+      #
+      # bad
+      # Component.new(classes: "mr-1")
+      #
+      # good
+      # Component.new(mr: 1)
+      class SystemArgumentInsteadOfClass < RuboCop::Cop::Cop
+        INVALID_MESSAGE = <<~STR
+          Avoid using CSS classes when you can use System Arguments.
```

Comment:
Perhaps we could include a link to the systems arguments docs in this message?

```suggestion
          Avoid using CSS classes when you can use System Arguments: https://primer.style/view-components/system-arguments.
```